src/Frontend-GNOME/Views/MessageTextView.cs

```diff
@@ -51,6 +52,7 @@ public class MessageTextView : Gtk.TextView
         private Gdk.Color    _MarkerlineColor = new Gdk.Color(255, 0, 0);
         private int          _MarkerlineBufferPosition;
         private int          _BufferLines = -1;
+        private IconCache    _EmojiCache = new IconCache("emoji");
```

Comment:
You can't version fields. If this is changed to a property later then you need to refactor all code that uses it (because of the different name)